### DISCUSSION TITLE
Branch comparison?

### DISCUSSION THREAD

**Discussion by aleksandersumowski** [extrawurst/gitui#1939] [Q&A] [answered]
Hi all,
Is it possible to use gitui to compare two branches? I'm thinking of some combination of `git log branch_1..branch_2`, `git diff branch_1 branch_2`, and `git show individual_commit`

**Comment by extrawurst** [comment, marked as answer]
sure open the branch popup and select which branch to compare with. it will show a full diff of your current state whatever branch you are on and the one you selected

**Reply by aleksandersumowski** [reply]
Found it! For the record:
B - opens branch popup
within the popup:
Shift-C opens comparison
